### PR TITLE
Fix/4947 workflow datetime field

### LINKED ISSUE #4947: Workflow: Date-time fields: inconsistency
#### Issue

A slight inconsistency in the workflow module when setting a condition based on date-time fields such as "Date Modified" between edit-view and detail-view:

In Edit-View I can choose only Year, Month, Day (in user-specific format).
In Detail-view I then see that it's actually taking the date I entered and adding time "00:00:00" to it.

Consequence:
When I set a condition "Date Modified" "Greater Than" "Value" "2018-01-15" in edit view, what the CRM actually does is "Date Modified" "Greater Than" "Value" "2018-01-15 00:00:00".

This means that actions taken on Jan 15 2018 are considered to meet the condition, while the user expects the condition only to be met on Jan 16 2018 and after.

#### Expected Behavior

Datetime fields should show in edit view as Date and time fields. User should be able to (and have to) choose date and time.

#### Actual Behavior

User can only choose the date part of a date-time field => SuiteCRM adds "00:00:00" to the date to create a date-time stamp.

#### Your Environment

* SuiteCRM Version used: 7.9.0

### PR DESCRIPTION
**Pull request by PedroErnst**
Makes datetime fields display as datetimecombo fields with hour and minute dropdowns.

## Description

Fixes #4947 

Previously in both AOW Workflow and AOR Reports condition fields of type datetime and datetimecombo were displayed as a simple input which allowed only to choose the date, but not the time.

screenshot from 2018-01-17 15-48-39

With this change the field has been expanded into datetimecombo field which allows more precise filtering.

screenshot from 2018-01-17 11-29-52

screenshot from 2018-01-17 15-39-05

screenshot from 2018-01-17 15-39-18

## How To Test This

1. Rebuild CSS
2. Go to either Reports or Workflow module.
3. Choose any related module
4. Set a condition based on a datetime field
5. See that the filters take into account the time

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

**Comment by Dillon-Brown** [comment]
Assessed

File: modules/AOW_WorkFlow/aow_utils.php [hunk 1]
```diff
@@ -585,6 +592,15 @@ function getModuleField($module, $fieldname, $aow_field, $view='EditView',$value
     return $ss->fetch($file);
 }
 
+/**
+ *  Convert a bracketed variable into a string that can become a JS variable
+ *
+ * @param $variable
```
Comment (daniel-samson) [inline modules/AOW_WorkFlow/aow_utils.php]: What is the type of $variable? Please put the type in the PHP doc. It helps our IDE's pick up on potential problems.

**Comment by PedroErnst** [comment]
@daniel-samson Updated my code

**Comment by Dillon-Brown** [comment]
Assessed

**Pull request merged**